docs/workWithSheet.md

```diff
@@ -0,0 +1,573 @@
+</googlespreadsheet.deleteSheetBatchRequest>
+```
+
+
+
+###### Properties
+
+* spreadsheetId :- Unique value of the spreadsheet
+* requests :- It contains data that is a kind of update to apply to a spreadsheet. To delete multiple sheets within the spread sheet, need to repeat `deleteSheet` property within the `requests` property.
+* fields [Optional] :- Specifying which fields to include in a partial response. For the following request only the `spreadsheetId` will be included in the response.
+
+
+
+###### Sample request
+
+Following is a sample request that can be used for deleteSheetBatchRequest operation. With the following request we can specify Id of the sheet need to be deleted.
+
+```json
+{
+    "clientId":"617729022812-xxxxxxxxxxxxxxxxxxxxx.apps.googleusercontent.com",
+    "clientSecret":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "refreshToken":"1/xxxxxxxxxxxxxxxxxxxxxxx-fCyxRTyf-LpK6fDWF9DgcM",
+    "accessToken":"ya29.xxxxxxxxxxxxxxxxxxxxxxxxxxxxx-pOuVvnbnHhkVn5u8t6Qr",
+    "apiUrl":"https://sheets.googleapis.com/v4/spreadsheets",
+    "spreadsheetId": "12KoqoxmykLLYbtsm6CEOggk5bTKMEIFGCD9EBdrXFGA",
+    "requests": [
+      {
+          "deleteSheet":
+          {
+            "sheetId": 813171540
+          }
+        }
+     ],
+      "fields": "spreadsheetId"
+}
+```
+
+
+
+###### Sample Response
+
+```json
+{
+    "spreadsheetId": "12KoqoxmykLLYbtsm6CEOggk5bTKMEIFGCD9EBdrXFGA"
+}
+```
+
+
+
+###### Related Google Spreadsheet documentation
+
+https://developers.google.com/sheets/reference/rest/v4/spreadsheets/request#deletesheetrequest
+
+
+
+#### Retrieve Sheet metadata
+
+This method can be used to acquire sheet properties and other metadata. If you only want to read the sheet properties, set the includeGridData query parameter to false to prevent the inclusion of the spreadsheet cell data. The Spreadsheet response contains an array of Sheet objects. The sheet titles and size information specifically can be found under the SheetProperties element of these objects.
+
+
+
+###### getSheetMetaData
+
+```xml
+<googlespreadsheet.getSheetMetaData>
+    <spreadsheetId>{$ctx:spreadsheetId}</spreadsheetId>
+    <includeGridData>{$ctx:includeGridData}</includeGridData>
+    <ranges>{$ctx:ranges}</ranges>
+    <fields>{$ctx:fields}</fields>
+</googlespreadsheet.getSheetMetaData>
+```
+
+
+
+###### Properties
+
+* spreadsheetId :- Unique value of the spreadsheet
+* includeGridData [Optional] :- True if grid data should be returned. This parameter is ignored if a field mask was set in the request.
+* fields [Optional] :- Specifying which fields to include in a partial response
+* ranges [Optional] :- The ranges to retrieve from the spreadsheet
+
+
+
+###### Sample request
+
+Following is a sample request that can be used for getSheetMetaData operation. With the following request we can specify spreadsheet Id, sheet details and the ranges to retrieve from the spreadsheet. Here the `Employees` is the sheet name and A1:B2 is the cell range.
```

Comment:
The following sample request calls the getSheetMetaData operation. Specify the spreadsheet Id, sheet details, and the ranges to retrieve from the spreadsheet. Here the `Employees` is the sheet name and A1:B2 is the cell range.